### DISCUSSION TITLE
[Document]: CN mirror site please

### DISCUSSION THREAD

**Discussion by charlzyx** [web-infra-dev/rsbuild#1498] [General]
### What problem does this feature solve?

我是cn的一个平平无奇的开发者，我有一个不成熟的小建议啊，能不能弄个location 在cn的文档网站，有时候想在手机上翻翻文档还要梯子真的很难受

### What does the proposed API look like?

rsbuild.cn
rspack.cn

**Comment by chenjiahan** [comment]
目前文档通过 Netlify 部署，在国内的访问速度不需要梯子的😂  有其他开发者遇到这个问题吗，可以在这个话题下留言

如果反馈较多，我们会考虑增加一个国内镜像

**Reply by charlzyx** [reply]
joke了 似乎是公司那边网不行
